content/blog/2021-12-21-December-21-community-gems.md

```diff
@@ -0,0 +1,174 @@
+want.
+
+### [If I have a DVC pipeline with many stages as well as `foreach` in every stage, is it possibile to run the whole pipeline but only for one element of the `foreach`?](https://discord.com/channels/485586884165107732/563406153334128681/915986804577026088)
+
+Another great question from @vgodie!
+
+If your stages look something like this for example:
+
+```yaml
+stages:
+  cleanups:
+    foreach: # List of simple values
+      - raw1
+      - labels1
+      - raw2
+    do:
+      cmd: clean.py "${item}"
+      outs:
+        - ${item}.cln
+  train:
+    foreach:
+      - epochs: 3
+        thresh: 10
+      - epochs: 10
+        thresh: 15
+    do:
+      cmd: python train.py ${item.epochs} ${item.thresh}
+```
+
+You should try the following command:
+
+```dvc
+$ dvc repro cleanups@labels1
+```
+
+This will run your whole pipeline, but only with `labels1` in the `cleanups`
+stage.
+
+### [Is it possible to pull experiments from the remote without checking out the base commit of those experiments?](https://discord.com/channels/485586884165107732/485596304961962003/910481311905505290)
+
+Thanks for the question @mattlbeck!
+
+You should be able to do this with `dvc exp pull origin exp-name`.
+
+If you have experiments with the same name on different commits, using
+`exp-name` won't work since it defaults to selecting the one based on your
+current commit if there are duplicate names.
+
+To work around this, you can use the full refname, like
+`refs/exps/e7/78ad744e8d0cd59ddqc65d5d698cf102533f85/exp-6cb7`, to specify the
+experiments that you want to work with.
+
+### [How should I handle checkpoints in PyTorch Lightning with DVCLive?](YouTube link here)
+
+This is a really good question that came from one of our Office Hours talks!
+Thanks [Ilia Sirotkin](https://www.linkedin.com/in/sirily/)!
+
+We have an [open issue](https://github.com/iterative/dvclive/issues/170) we
+encourage you to follow for more details and to even contribute!
+
+Python Lightning handles checkpoints differently from other libraries. This
+affects the way metrics logging is executed and how models are saved.
+
+You can write a custom callback to control saving everything and track it with
+DVC and this is the workaround we suggest. You can implement the
+`after_save_checkpoint` method and save the model file.
+
+### [Is there a feature for DVC to only sample and cache a subset of the tracked dataset?](https://discord.com/channels/485586884165107732/485596304961962003/917778575845900340)
+
+Really great question @Abdi!
+
+You should be able to use the streaming capability of the DVC API to achieve
+this goal.
+
+Here is an exaple of a Python script that would do this:
+
+```python
+from dvc.api import open as dvcopen
+
+with dvcopen('data','{repo_url}') as fd:
+    for line in fd:
+        print(line)
+```
+
+---
+
+https://media.giphy.com/media/h5Ct5uxV5RfwY/giphy.gif
+
+At our January Office Hours Meetup we will be doing a new thing!
```

Comment:
Can we share a little more about the "new thing"? As a reader, I'd like to see at least some details.

Reply:
No problem! This is just some placeholder text from earlier. 😅  I usually update this once the Meetup link is up. But it's updated now!